jlibra-core/src/main/java/dev/jlibra/move/Move.java

```diff
@@ -1,18 +1,38 @@
 package dev.jlibra.move;
 
-import com.google.protobuf.ByteString;
-
-import org.bouncycastle.util.encoders.Hex;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
 
 public class Move {
 
-    private static final String peerToPeerTransferBinary =
-            "4c49425241564d0a010007014a00000004000000034e000000060000000c5" +
-            "4000000060000000d5a000000060000000560000000290000000489000000" +
-            "2000000007a90000000e00000000000001000200010300020002040200030" +
-            "003020402063c53454c463e0c4c696272614163636f756e74046d61696e0f" +
-            "7061795f66726f6d5f73656e6465720000000000000000000000000000000" +
-            "0000000000000000000000000000000000001020104000c000c0111010002";
+    public static byte[] peerToPeerTransferAsBytes() {
+        try {
+            InputStream jsonCode = Move.class.getResourceAsStream("/move/peer_to_peer_transfer.bin.json");
+            String json = readFullyAsString(jsonCode, "UTF-8");
+            String[] bytesAsString = json.substring(json.indexOf("[")+1, json.indexOf("]")).split(",");
+            byte[] bytes = new byte[bytesAsString.length];
+            int idx = 0;
+            for (String byteAsString : bytesAsString) {
```

Comment:
You're converting the string to Integer, converting that to byte value, then doing that & 0xFF -trick and then casting the thing to byte.
But it should be enough just to cast the int to byte like `(byte) Integer.parseInt(byteAsString)`

Reply:
Maybe this could be an util runnable class with main method which takes input file as a parameter and produces hex string which we could use to replace the value for p2p script bin.
This way we could keep this class clean and still achieve the same thing. I agree with @ketola that it is a lot of code.
I don't like dependency on file resource.

Reply:
I restructured it with your comments - I think I don't get it shorter than that, but it looks way better now. Didn't know about the (byte)Integer - that's waay better than my stackoverflow "solution" :)